### PR TITLE
Added volume slider and set the audio session category to ambient so …

### PR DESCRIPTION
**Pull request by thales17**
…other audio can be mixed with the emulator audio

### CODE REVIEW

File: Provenance/Emulator/PVEmulatorViewController.m [hunk 1]
```diff
@@ -213,7 +213,7 @@ - (void)viewDidLoad
     [self.emulatorCore startEmulation];
 
     self.gameAudio = [[OEGameAudio alloc] initWithCore:self.emulatorCore];
-    [self.gameAudio setVolume:1.0];
+    [self.gameAudio setVolume:[[NSUserDefaults standardUserDefaults] floatForKey:@"volume"]];
```
Comment (jasarien) [inline Provenance/Emulator/PVEmulatorViewController.m]: Use the `PVSettingsModel` class for storing settings.

File: Provenance/PVAppDelegate.m [hunk 2]
```diff
@@ -19,6 +19,7 @@ @implementation PVAppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
     [[UIApplication sharedApplication] setIdleTimerDisabled:[[PVSettingsModel sharedInstance] disableAutoLock]];
+    [[NSUserDefaults standardUserDefaults] registerDefaults:@{@"volume" : @1.0}];
```
Comment (jasarien) [inline Provenance/PVAppDelegate.m]: Use `PVSettingsModel`

File: Provenance/Settings/PVSettingsViewController.m [hunk 3]
```diff
@@ -33,6 +33,8 @@ - (void)viewDidLoad
     [self.dPadDeadzoneSlider setValue:[settings dPadDeadzoneValue]];
 	[self.autoLockSwitch setOn:[settings disableAutoLock]];
     [self.vibrateSwitch setOn:[settings buttonVibration]];
+    [self.volumeSlider setValue:[[NSUserDefaults standardUserDefaults] floatForKey:@"volume"]];
+    [self.volumeValueLabel setText:[NSString stringWithFormat:@"%.0f%%", self.volumeSlider.value * 100]];
```
Comment (jasarien) [inline Provenance/Settings/PVSettingsViewController.m]: `PVSettingsModel` (noticing a pattern here) ;)

**Comment by thales17** [comment]
Good feedback!, I have made those changes.

**Pull request merged**